src/@ionic-native/plugins/firebase/index.ts

```diff
@@ -30,89 +30,108 @@ import { Observable } from 'rxjs/Observable';
   plugin: 'cordova-plugin-firebase',
   pluginRef: 'FirebasePlugin',
   repo: 'https://github.com/arnesson/cordova-plugin-firebase',
-  platforms: ['Android', 'iOS']
+  platforms: ['Android', 'iOS'],
 })
 @Injectable()
 export class Firebase extends IonicNativePlugin {
-
   /**
    * Get the device token
    * @return {Promise<any>}
    */
   @Cordova()
-  getToken(): Promise<any> { return; }
+  getToken(): Promise<any> {
+    return;
+  }
 
   /**
    * Get notified when a token is refreshed
    * @return {Observable<any>}
    */
   @Cordova({
-    observable: true
+    observable: true,
   })
-  onTokenRefresh(): Observable<any> { return; }
+  onTokenRefresh(): Observable<any> {
+    return;
+  }
 
   /**
    * Get notified when the user opens a notification
    * @return {Observable<any>}
    */
   @Cordova({
-    observable: true
+    observable: true,
```

Comment:
Why the added `,` here?

Reply:
My apologies, TSLint has a config option on by default that adds a trailing comma, so when you need to extend an object or array, you can just start typing. I've grown used to it because I'm lazy :)